node/cli/src/chain_spec/turing.rs

```diff
@@ -0,0 +1,232 @@
+// 	// 				get_account_id_from_seed::<sr25519::Public>("Bob//stash"),
+// 	// 				get_account_id_from_seed::<sr25519::Public>("Charlie//stash"),
+// 	// 				get_account_id_from_seed::<sr25519::Public>("Dave//stash"),
+// 	// 				get_account_id_from_seed::<sr25519::Public>("Eve//stash"),
+// 	// 				get_account_id_from_seed::<sr25519::Public>("Ferdie//stash"),
+// 	// 			],
+// 	// 			id,
+// 	// 		)
+// 	// 	},
+// 	// 	vec![],
+// 	// 	None,
+// 	// 	None,
+// 	// 	None,
+// 	// 	Extensions {
+// 	// 		relay_chain: "rococo-local".into(), // You MUST set this to the correct network!
+// 	// 		para_id: id.into(),
+// 	// 	},
+// 	// )
+// }
+
+pub fn turing_local_testnet_config() -> Result<ChainSpec, String>  {
+	local_testnet_config(TURING_PARA_ID)
+}
+
+fn local_testnet_config(id: ParaId) -> Result<ChainSpec, String>  {
+	ChainSpec::from_json_bytes(&include_bytes!("../../res/oak-testnet.json")[..])
+}
+
+
+// pub fn local_testnet_config(id: ParaId) -> ChainSpec {
+// 	// Give your base currency a unit name and decimal places
+// 	let mut properties = sc_chain_spec::Properties::new();
+// 	properties.insert("tokenSymbol".into(), "TUR".into());
+// 	properties.insert("tokenDecimals".into(), 10.into());
+// 	properties.insert("ss58Format".into(), 42.into());
+
+// 	ChainSpec::from_genesis(
+// 		// Name
+// 		"Local Testnet",
+// 		// ID
+// 		"local_testnet",
+// 		ChainType::Local,
+// 		move || {
+// 			testnet_genesis(
+// 				// initial collators.
+// 				vec![
+// 					(
+// 						get_account_id_from_seed::<sr25519::Public>("Alice"),
+// 						get_collator_keys_from_seed("Alice"),
+// 					),
+// 					(
+// 						get_account_id_from_seed::<sr25519::Public>("Bob"),
+// 						get_collator_keys_from_seed("Bob"),
+// 					),
+// 				],
+// 				vec![
+// 					get_account_id_from_seed::<sr25519::Public>("Alice"),
+// 					get_account_id_from_seed::<sr25519::Public>("Bob"),
+// 					get_account_id_from_seed::<sr25519::Public>("Charlie"),
+// 					get_account_id_from_seed::<sr25519::Public>("Dave"),
+// 					get_account_id_from_seed::<sr25519::Public>("Eve"),
+// 					get_account_id_from_seed::<sr25519::Public>("Ferdie"),
+// 					get_account_id_from_seed::<sr25519::Public>("Alice//stash"),
+// 					get_account_id_from_seed::<sr25519::Public>("Bob//stash"),
+// 					get_account_id_from_seed::<sr25519::Public>("Charlie//stash"),
+// 					get_account_id_from_seed::<sr25519::Public>("Dave//stash"),
+// 					get_account_id_from_seed::<sr25519::Public>("Eve//stash"),
+// 					get_account_id_from_seed::<sr25519::Public>("Ferdie//stash"),
+// 				],
+// 				id,
+// 			)
+// 		},
+// 		// Bootnodes
+// 		vec![],
+// 		// Telemetry
+// 		None,
+// 		// Protocol ID
+// 		Some("template-local"),
+// 		// Properties
+// 		Some(properties),
+// 		// Extensions
+// 		Extensions {
+// 			relay_chain: "rococo-local".into(), // You MUST set this to the correct network!
+// 			para_id: id.into(),
+// 		},
+// 	)
+// }
+
+// fn testnet_genesis(
```

Comment:
Does this chain_spec/turing.rs need to be cleaned up? Do we still need those commented code?

Reply:
I plan on cleaning it up in the next PR with parachain refactor